tests/Functional/RoutingTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NewsBundle\Tests\Functional\Routing;
+
+use Sonata\NewsBundle\Tests\App\AppKernel;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
```

Comment:
Thank you for the credit.

Reply:
I copy your work, delete this will be rude.